lib/index.js

```diff
@@ -534,11 +534,9 @@ var parseInputUnions = function parseInputUnions(_ref8) {
 
 			typesCache[schemaId] = new _graphqlUnionInputType2.default({
 				name: extractTypeName(context) + 'Input',
-				types: function types() {
-					return schema.anyOf.map(function (subSchema) {
-						return typesCache[subSchema.$$inputType];
-					});
-				},
+				inputTypes: schema.anyOf.map(function (subSchema) {
```

Comment:
@wrak Do we need to send `typeName: 'CompanyInput'` with the `Input` suffix as we discussed? Consider using second option from https://github.com/wrak/graphql-union-input-type#optionsobject to allow sending just `typeName: 'Company'` so BE devs are not confused.